### PR TITLE
correct styling for language bar

### PR DESCRIPTION
**Pull request by Rizziepit**

### CODE REVIEW

**Comment by Rizziepit** [comment]
@miltontony ready for review. Trivial changes.

File: springboard_iogt/templates/base.jinja2 [hunk 1]
```diff
@@ -21,9 +21,11 @@
                 <span>
                     <a href="{{ 'locale_matched'|route_url(language=lang_code) }}">{{ lang_code|display_language_name }}</a>
                 </span>
-                <span> | </span>
+                {% if not loop.last %}<span> | </span>{% endif %}
             {% endfor %}
-            <span><a href="{{ 'locale_change'|route_url }}">&hellip;</a></span>
+            {% if display_languages|length + 1 < available_languages|length %}
```
Comment (miltontony) [inline springboard_iogt/templates/base.jinja2:26]: why `length + 1` here?

Comment (Rizziepit) [inline springboard_iogt/templates/base.jinja2:26]: `display_languages` is `featured_languages` minus the currently selected language. So if your currently selected language is not in the `featured_languages` list, `display_languages` = `featured_languages`.

Comment (miltontony) [inline springboard_iogt/templates/base.jinja2:26]: gotcha :ok_hand:

**Comment by miltontony** [comment]
:+1:

**Pull request merged**